numpy/lib/npyio.py

```diff
@@ -1357,7 +1357,7 @@ def genfromtxt(fname, dtype=float, comments='#', delimiter=None,
                missing_values=None, filling_values=None, usecols=None,
                names=None, excludelist=None, deletechars=None,
                replace_space='_', autostrip=False, case_sensitive=True,
-               defaultfmt="f%i", unpack=None, usemask=False, loose=True,
+               defaultfmt="f%i", unpack=None, ndmin=0, usemask=False, loose=True,
```

Comment:
Should be added at the end I think, in case someone is crazy enough to pass in all arguments as non-keyword arguments. Or maybe it is crazy enough to not have to care, hehe.